.gitignore

```diff
@@ -0,0 +1 @@
+.idea/
```

Comment:
I know that this repo contains only constants, but would be nice to ignore `node_modules/` by default as well